### ISSUE TITLE
"C-c z is undefined"

### ISSUE THREAD

**Issue by mobatmedia** [Vidianos-Giannitsis/zetteldesk.el#10]
After following your [sample config instructions](https://github.com/Vidianos-Giannitsis/zetteldesk.el#sample-config), I press C-c z and get "C-c z" is undefined. Yet, your documentation says "After setting these, just press C-c z and explore the hydra." What key step am I missing in your installation set up instructions?

Emacs: "GNU Emacs 27.2 (build 1, x86_64-w64-mingw32) of 2021-03-26"
  org                9.5.5         installed             Outline-based notes management and organizer
  org-gtd            20220213.41   installed             An implementation of GTD.
  org-roam           20220908.1625 installed             A database abstraction layer for Org-mode
  helm               20221117.1800 installed             Helm is an Emacs incremental and narrowing framework
  hydra              20220910.1206 installed             Make bindings that stick around.
  pretty-hydra       20210221.834  installed             A macro for creating nice-looking hydras
  python-mode        20221031.1030 installed             Python major mode
  use-package        20221117.1610 installed             A configuration macro for simplifying your .emacs
  zetteldesk         20220703.1648 installed             A revision and outlining tool for org-roam
  zetteldesk-info    20220626.1100 installed             A zetteldesk extension for interacting with the info program
  zetteldesk-kb      20221009.2122 installed             Keybindings for zetteldesk.el
  zetteldesk-ref     20220619.2028 installed             A zetteldesk extension for interfacing with literature nodes
  zetteldesk-remark  20220626.1100 installed             Org-Remark integration for zetteldesk.el

**Comment by Vidianos-Giannitsis** [comment]
You are indeed correct. I replicated what you said in "emacs -Q" and it did give me that error. I am not certain why that gave an error as I recall testing it when I set it up. However, since I don't use that part myself (I define the keybinding using general.el to fit the general scheme of my config) I wasn't aware it was broken.

The above commit should fix the issue. Following the same steps with an "emacs -Q" I no longer get the error and the hydra is launched properly. Do try it out yourself and tell me.
